Fireworks/database.py

```diff
@@ -186,21 +186,21 @@ def __repr__(self):
 
     return SimpleTable
 
-class DBSource(Source):
+class DBPipe(Pipe):
     """
-    Source that can iterate through the output of a database query.
+    Pipe that can iterate through the output of a database query.
     """
     def __init__(self, table, engine, query = None, columns_and_types= None):
         """
         Args:
             table (sqlalchemy.ext.declarative.api.DeclarativeMeta): Table to perform query on. You can alternatively provide the name of the
                 table as a string, and the schema will be extracted from the engine.
             engine (sqlalchemy.engine.base.Engine): Engine correspondign to the database to read from.
-            query: Can optionally provide an SQLalchemy Query object. If unspecified, the DBSource will perform a SELECT * query.
+            query: Can optionally provide an SQLalchemy Query object. If unspecified, the DBPipe will perform a SELECT * query.
         """
+        Pipe.__init__(self)
         self.engine = engine
         Session = sessionmaker(bind=engine)
```

Comment:
Same here DBPipe shouldn't be managing the session let that be an input when initiating or query using the engine directly.